### ISSUE TITLE
[a11y-defect-3]: `aria-labels` in VA table pagination are not announced by NVDA screen reader

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#95248]
### Point of contact

Trevor Pierce

### Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### Details

I discovered the [VA table with pagination](https://design.va.gov/components/table#borderless-with-pagination) did not announce the `aria-labels` when those labels were applied to non-interactive elements including divs and list items. This was observed using NVDA on Chrome and Firefox on Windows 11. 

This can lead to an incomplete understanding of the pagination controls for screen reader users. Screen shot attached below showing the elements I observed this behavior.

---

![Image](https://github.com/user-attachments/assets/c0b9c3fc-0115-4a01-b039-a74c77949b2a)

### Reproduction steps

1. Navigate to a view with a VA table with pagination. Ideally one that has the ellipsis for more pages than can be displayed.
2. Turn on a screen reader (preferably NVDA)
3. Traverse through the pagination links using `DOWN_ARROW` to move node to node
4. Verify the ellipsis is ignored and the "Next page" accessible label is ignored in favor of "Next, link"

### Proposed solution or next steps

I'd propose the ellipsis code be refactored to use an SR-only text block inside the LI instead of an `aria-label`. I'd also ask that we consider removing the accessible labels from the next and previous links and renaming them "Next | Previous page".

```html
<li class="usa-pagination__item usa-pagination__overflow va-pagination__item">
  <span class="vads-u-visibility--screen-reader">Ellipsis indicating non-visible pages</span>
  <span aria-hidden="true">…</span>
</li>
```

### References, articles, or WCAG support

1. Understanding [SC 2.4.4](https://w3.org/TR/WCAG22#link-purpose-in-context): Link Purpose (In Context) (Level A)
2. [Aria-Label is Not Always the Answer](https://eevis.codes/blog/2021-11-29/aria-label-is-not-always-the-answer/#aria-label-and-non-supported-elements)

### Type of issue

- [ ] Axe-core
- [x] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [x] Link and button labels
- [ ] Semantic HTML
- [x] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else

**Comment by dysbo** [comment]
Per @jstrothman this does appear to be the current version of the component.  @1Copenut to investigate and possibly deliver changes to Design System component and/or create a ticket to track that work in that project.

**Comment by 1Copenut** [comment]
Confirmed this markup pattern is in use on the current  [VA pagination component](https://design.va.gov/components/pagination). Going to check for an existing Design System issue and explore a code patch if none exists.
